### PR TITLE
upstream update to fusion46 (v0.9.7)

### PR DESCRIPTION
**Pull request by islandbitcoin**

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 id: lnbits 
 title: "LNBits"
-version: 0.9.5.1
-release-notes: "Initial Release for embassyOS"
+version: 0.9.7
+release-notes: | 
+  * Fusion46 [Release Notes](https://github.com/lnbits/lnbits/releases/tag/0.9.6.1)
```
Comment (chrisguida) [inline manifest.yaml]: Why no release notes for 0.9.7?

Comment (islandbitcoin) [reply manifest.yaml]: very strange to me too - the release notes link in 0.9.7 links to 0.9.6.1 ... like they didn't bother making another one, just linked to the same one as the full release notes

Comment (islandbitcoin) [reply manifest.yaml]: I could link it to https://github.com/lnbits/lnbits/releases/tag/0.9.7, but that's basically empty

Comment (islandbitcoin) [reply manifest.yaml]: i'll put both :-)

File: manifest.yaml [hunk 2]
```diff
@@ -1,7 +1,10 @@
 id: lnbits 
 title: "LNBits"
-version: 0.9.5.1
-release-notes: "Initial Release for embassyOS"
+version: 0.9.7
+release-notes: | 
+  * Fusion46
```
Comment (chrisguida) [inline manifest.yaml]:
```suggestion
  * Update to new upstream version Fusion46 (v0.9.7)
```

**Review by chrisguida — CHANGES_REQUESTED** [review]
See suggested change

**Review by chrisguida — APPROVED** [review]
Seems fine

**Pull request merged**